src/mongocrypt-crypto.c

```diff
@@ -1280,6 +1283,9 @@ DECLARE_ALGORITHM (FLE2AEAD, CTR, SHA_256)
 // FLE2 used with ESC/ECOC tokens: AES-256-CTR no HMAC
 DECLARE_ALGORITHM (FLE2, CTR, NONE)
 
+// FLE2 used with FLE2IndexedEncryptedValue: AES-256-CBC HMAC/SHA-256
+DECLARE_ALGORITHM (FLE2v2, CBC, SHA_256)
```

Comment:
```suggestion
DECLARE_ALGORITHM (FLE2v2AEAD, CBC, SHA_256)
```

Since this algorithm includes HMAC.

Reply:
I had a premonition this request would come.
Renamed and moved throughout.